ts/theme/variables.ts

```diff
@@ -39,24 +39,30 @@ const customVariables = Object.assign(materialVariables, {
   brandLightGray: "#E6E9F2",
   brandDarkGray: "#5C6F82",
 
+  fontColor: "#17324D",
```

Comment:
`fontColor` is a little generic, is this the default color for all the text in the app?